css/secondary-page.styl

```diff
@@ -105,6 +115,33 @@
       overflow: auto
       padding: 1em
 
+&.has-project-context
+  background-color: transparent !important
+
+  .hero
+    background-image: none !important
+    background-color: transparent !important
+    padding-top: 2vw !important
+    min-height: initial !important
+
+    .hero-container
+      margin: 0 !important
+      max-width: initial !important
+
+      h1
+        text-align: center
+    .overlay
+      background-image: none !important
+      background-color: transparent !important
```

Comment:
If you instead scope this under `on-seconday-page`, all of the `!important`s can be removed and the styles apply correctly.